packages/oui-dual-list/src/dual-list.directive.js

```diff
@@ -0,0 +1,24 @@
+import controller from "./dual-list.controller";
+import template from "./dual-list.html";
+
+export default () => ({
```

Comment:
Prefer using angular component instead of a directive, since it will be use with `<oui-dual-list></oui-dual-list>`.
In this case, the file will renamed `dual-list.component.js` :)